ios/Classes/Blemulator/Model/Peripheral.h

```diff
@@ -0,0 +1,13 @@
+@interface Peripheral : NSObject
```

Comment:
Was it not possible to reuse the class from MBA?

Reply:
It's a part of RxBluetoothKit and it's not exposed to objc (would require some changes in RxBluetoothKit)